sky/templates/vsphere-ray.yml.j2

```diff
@@ -0,0 +1,108 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.provision.vsphere
+  region: {{region}}
+  availability_zone: {{zones}}
+
+auth:
+  ssh_user: ecl
+  ssh_private_key: {{ssh_private_key}}
+
+available_node_types:
+  ray.head.default:
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+{% if num_nodes > 1 %}
+  ray_worker_default:
+    min_workers: {{num_nodes - 1}}
+    max_workers: {{num_nodes - 1}}
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+{%- endif %}
```

Comment:
These part is no longer needed, as our new provisioner will not use the worker config.

Reply:
Removed